armonik-versions.txt

```diff
@@ -1,5 +1,5 @@
 infra=0.0.2
-core=0.13.2
-worker=0.11.1
+core=0.14.2
```

Comment:
Is version 0.14.0 or 0.14.2 for Core? (see changlog)

Reply:
0.14.2

Reply:
OK we must update changelog